include/vcpkg/base/files.h

```diff
@@ -367,7 +384,7 @@ namespace vcpkg
 #endif // _WIN32
 
 #if defined(_WIN32)
-    path win32_fix_path_case(const path& source);
+    Path win32_fix_path_case(const Path& source);
```

Comment:
```suggestion
    Path win32_fix_path_case(Path source);
```
to support in-place modification